db/migrate/20171031010000_add_schema_migrations_ran.rb

```diff
@@ -10,7 +10,7 @@ def up
 
     pglogical = ActiveRecord::Base.connection.pglogical
     if pglogical.enabled? && pglogical.replication_sets.include?('miq')
-      pglogical.replication_set_add_table('miq', "schema_migrations_ran", true)
+      pglogical.replication_set_add_table('miq', "schema_migrations_ran")
```

Comment:
As discussed, I'd prefer if we explicitly passed `false` here, and maybe add a comment as well to declare our intention.